_posts/2020-10-16-kaitai-struct-v0.9-released.md

```diff
@@ -99,12 +101,19 @@ fixes a lot of bugs and includes quite a few infrastructure improvements.
     Gradle plugin](https://github.com/valery1707/kaitai-gradle-plugin)
 * Infrastructure updates:
   * Unstable binary builds are available for all platforms after every CI build at Bintray ([#63](https://github.com/kaitai-io/kaitai_struct/issues/63))
-  * KSY language reference replaced with [documentation](https://doc.kaitai.io/ksy_diagram.html) generated from JSON schema
+  * KS language reference replaced with [documentation](https://doc.kaitai.io/ksy_diagram.html) generated from [the JSONSchema](https://github.com/kaitai-io/ksy_schema)
   * [https://formats.kaitai.io/](https://formats.kaitai.io/) is rebuilt automatically with CI/CD
   * Brand new modular CI/CD system for compiler, underlying
     CI-agnostic, working on multiple different OSes in parallel
     (Linux, Windows, macOS) and showing status at [https://ci.kaitai.io/](https://ci.kaitai.io/)
   * Generate test assertion specs from language-agnostic [KST specs](https://doc.kaitai.io/kst.html)
+* Related projects:
+  * Added an [official library of compression algorithms](https://github.com/kaitai-io/kaitai_compress) to be used in `process`.
+  * Created [a list of ![awesome](https://camo.githubusercontent.com/1997c7e760b163a61aba3a2c98f21be8c524be29/68747470733a2f2f617765736f6d652e72652f62616467652e737667) projects related to Kaitai Struct](https://github.com/kaitai-io/awesome-kaitai)
```

Comment:
Please keep the imperative style (add!, support!, allow!, implement!, ...) of the release notes. 

```suggestion
  * Add an [official library of compression algorithms](https://github.com/kaitai-io/kaitai_compress) to be used in `process`.
  * Create [a list of ![awesome](https://camo.githubusercontent.com/1997c7e760b163a61aba3a2c98f21be8c524be29/68747470733a2f2f617765736f6d652e72652f62616467652e737667) projects related to Kaitai Struct](https://github.com/kaitai-io/awesome-kaitai)
```